### DISCUSSION TITLE
CSS, enter on value end, can it keep semi-colon and put cursor on new line?

### DISCUSSION THREAD

**Discussion by benfrain** [hrsh7th/nvim-cmp#2082] [Q&A]
I'm asking the same question here and elsewhere so thought it might be more useful written up as a blog post with some example videos. But in short, can nvim-cmp be configured so that pressing enter on a value keeps the semi-colon but moves to a new line?

https://benfrain.com/in-search-of-the-perfect-autocomplete-for-css/

**Comment by hrsh7th** [comment]
this is not part of auto-completion. but I can recommend to use [nvim-insx](https://github.com/hrsh7th/nvim-insx) for this situation.
(of course, you must setup nvim-insx collectly).
